ice-adapter/src/main/java/com/faforever/iceadapter/ice/PeerIceModule.java

```diff
@@ -447,12 +448,10 @@ void close() {
         if(turnRefreshModule != null) {
             turnRefreshModule.close();
         }
-        if(connectivityChecker != null) {
-            connectivityChecker.stop();
-        }
         if(agent != null) {
             agent.free();
         }
+        connectivityChecker.stop();
```

Comment:
What if connectivity checker is null?
It's only running on the offer side. (at least it used to)

Reply:
I believe I made this change because the data flow analysis said this could never be null

Reply:
In fact it is declared at field declaration as a final field so it will never be null